src/domain/products/components/prices-form/nested-price.tsx

```diff
@@ -47,7 +47,7 @@ const NestedPrice = ({ form, nestedPrice }: Props) => {
             <span className="inter-base-regular text-grey-50">
               {currencies[currencyPrice.currency_code.toUpperCase()].name}
             </span>
-            <IncludesTaxTooltip />
+            <IncludesTaxTooltip includesTax={currencyPrice?.includes_tax}/>
```

Comment:
```suggestion
            <IncludesTaxTooltip includesTax={currencyPrice.includes_tax}/>
```

nit: no reasons to use `?` notation as the currencyPrice will exist at this point